api-clients/HttpClientConsoleApp/HttpClientConsoleApp/Entities/Supplier.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace HttpClientConsoleApp.Entities;
+
+public class Supplier
```

Comment:
Actually, do you need this class if it has no any properties? It will be omitted by your serializer if JSON-model has property, but object not